gettsim/taxes/zve.py

```diff
@@ -147,6 +150,7 @@ def kinderfreibetrag(tax_unit, params, kindergeld_params):
 
 def zve_nokfb(tax_unit, params):
```

Comment:
Maybe we should strart giving `nokfb` a german descriptive name. Like `kein_kind_freib

Reply:
OK, but I did not introduce `nokfb/kfb` here so might be better suited in a large rename PR.